### PR TITLE
Re-enable LA reports for live release

### PR DESCRIPTION
**Pull request by tgreyuk**

### CODE REVIEW

**Comment by tgreyuk** [comment]
[Disable functionality not yet ready for live - again](https://trello.com/c/uIP2Rk94/314-disable-functionality-not-yet-ready-for-live-again)

File: src/app/features/reports/pages/reports/reports.component.html [hunk 1]
```diff
@@ -33,5 +33,26 @@ <h2 *ngIf="isParent" class="govuk-body-l">Workforce Development Fund</h2>
         </div>
       </div>
     </app-details>
+
+    <ng-container *ngIf="canRunLocalAuthorityReport || isAdmin">
```
Comment (omarCreativeDev) [inline src/app/features/reports/pages/reports/reports.component.html]: is the `isAdmin` check needed here? im guessing `canRunLocalAuthorityReport` could be true whilst user isn't an admin? ideally the permission should take care of both in my opinion

Comment (tgreyuk) [reply src/app/features/reports/pages/reports/reports.component.html]: Not sure, but i would rather take it out on dev than here as this is the code that has been tested against (this is going straight into live).

Comment (tgreyuk) [reply src/app/features/reports/pages/reports/reports.component.html]: Actually i think canRunLocalAuthorityReport has some other conditions set on it.

**Pull request merged**